articles/active-directory-b2c/faq.yml

```diff
@@ -220,6 +220,16 @@ sections:
         answer: |
           No, Azure AD B2C is a pay-as-you-go Azure service and is not part of Enterprise Mobility Suite.
           
+       - question: |
+          Can I purchase Azure AD Premium P1 and Azure AD Premium P2 licensing for my Azure AD B2C tenant?
+        answer: |
+          No, Azure AD B2C tenants do not support Azure AD Premium P1 or Azure AD Premium P2 licensing.  Azure AD B2C tenants natively support some similar features as Azure AD Premium see [Supported Azure AD features](supported-azure-ad-features.md) but not all of the same features.  Additionally, Azure AD B2C tenants can use [External Identity Premium P1 and P2 billing models](billing.md) but these are not the same as Azure AD Premium P1 and P2 licensing for a standard Azure AD tenant.
```

Comment:
Can we add/update this information in [Supported Azure AD features](https://docs.microsoft.com/en-us/azure/active-directory-b2c/supported-azure-ad-features)? 

What do you think?

Reply:
I added it there as well, but think it should be in FAQ as well.  It's a very frequently asked question.